packages/react-component-library/src/components/TopLevelNavigation/Sheet.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
```

Comment:
Do these Sheet related sub-components warrant being in a sub-directory of their own?